Package.swift

```diff
@@ -37,7 +37,7 @@ let package = Package(
         ),
     ],
     dependencies: [
-        .package(name: "Firebase", url: "https://github.com/firebase/firebase-ios-sdk.git", "7.3.1"..<"9.0.0"),
+        .package(name: "Firebase", url: "https://github.com/firebase/firebase-ios-sdk.git", .exact("10.0.0")),
```

Comment:
You should make it greater than 7.3.1 and less than 11.

Reply:
Made the change. Thanks.